### ISSUE TITLE
Some examples are broken

### ISSUE THREAD

**Issue by MoeweX** [stratosphere/stratosphere#894]
Hi,

I am currently enhancing the webclient (#711) and I noticed, that some examples seem to be broken. For example the Kmeans.jar.
![bildschirmfoto 2014-06-02 um 13 29 38](https://cloud.githubusercontent.com/assets/5738978/3146540/3d4423ec-ea49-11e3-9617-a3f6ba962f3e.png)
If I try to draw this one and others it does not work like it should. My ajax request to the intern server is not successful because of an exception...: 

![bildschirmfoto 2014-06-02 um 13 33 16](https://cloud.githubusercontent.com/assets/5738978/3146568/b7d17d26-ea49-11e3-9d1f-eed0fe2e8a22.png)

**Comment by rmetzger** [comment]
I think this exception does not belong to the KMeans example.
The exception is thrown in the scala wordcount example

**Comment by StephanEwen** [comment]
Yes. The issue is the following: In order to get a plan from the programs,
they need to be runnable without parameters. What you see here is the
exception that the program throws if it does not get its parameters.

I am not sure how we can solve that. If a program does not want to proceed,
then it does not ;-)

You should still get the Optimized plan when you supply proper parameters.
